.github/workflows/linkinator.yml

```
@@ -0,0 +1,18 @@
+on:
+  push:
+    branches:
+      - $default-branch
+  pull_request:
+  workflow_dispatch:
+name: Linkinator CI
+jobs:
+  linkinator:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: JustinBeckwith/linkinator-action@v1
+        with:
+          paths: "**/*.md"
+          markdown: true
+          retry: true
+          verbosity: 'ERROR'
```

```suggestion
          verbosity: 'ERROR'
          linksToSkip: "https://www.linode.com/docs/"
```

Based on doc here: https://github.com/JustinBeckwith/linkinator-action